exercises/colorsquare.js

```diff
@@ -54,7 +64,15 @@ class ColorSquare{
 			make sure the rightNeighbor is something!  the rightmost element won't have a neighbor
 		*/
 	handleClick(){
-
+		if (this.colorArray[this.colorIndex + 1]){
+			this.colorIndex += 1;
+		}else if (this.colorArray.length){
```

Comment:
this would only be not true if the length was 0, which would mean the array was empty to begin with.  If you did pass in an empty array, what would happen?